integration-tests/spring-data-jpa/src/it/spring-configuration/verify.groovy

```diff
@@ -0,0 +1,8 @@
+//Check that file exits
+String base = basedir
+File mvnInvokerLog = new File(base, "build.log")
+assert mvnInvokerLog.exists()
+
+def contentFile = mvnInvokerLog.text
+assert contentFile.contains("Quarkus does not support the spring.jpa.show-sql property you may try to use the Quarkus equivalent one : quarkus.hibernate-orm.log.sql.spring.jpa.show-sql property.")
+assert contentFile.contains("Quarkus does not support the spring.jpa.data.hibernate.dialect property.")
```

Comment:
Why don't we have the equivalent here?

Reply:
We have it. I would like to test the case of a Spring Boot property that we have not and take this one but it doesn't really exist. I've done it with a truly one `spring.jpa.open-in-view` for which we don't have a quarkus equivalent.